include/RED4ext/Scripting/CProperty.hpp

```diff
@@ -15,7 +15,18 @@ struct CProperty
     CName group;
     CClass* parent;
     uint32_t valueOffset;
-    uint64_t flags;
+
+    struct Flags
+    {
+        uint64_t b0 : 9;
+        uint64_t isOut : 1;
+        uint64_t isOptional : 1;
+        uint64_t b12 : 9;
+        uint64_t b21 : 1;
```

Comment:
This is not `0x200000`, it is on bit 20 instead of 21.

Reply:
F. Counting is hard.